src/app/shared/object-list/my-dspace-result-list-element/pool-search-result/pool-search-result-list-element.component.ts

```diff
@@ -85,6 +101,14 @@ export class PoolSearchResultListElementComponent extends SearchResultListElemen
       followLink('submitter')
     ), followLink('action'));
 
+    // Get configuration for duplicate detection feature
+    this.enableDetectDuplicates$ = this.configService.findByPropertyName('duplicate.enable').pipe(
```

Comment:
This variable doesn't appear to be used anywhere?  I'm confused why you set this variable but then never seem to use it.  (Maybe I'm missing where it is used?)

Reply:
removed